src/app/custom-navigation-mode/custom-navigation-mode-factory.provider.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseNavigationModeFactory, WizardComponent, NavigationMode } from 'angular-archwizard';
+import { CustomNavigationMode } from './custom-navigation-mode';
+
+export class CustomNavigationModeFactory extends BaseNavigationModeFactory {
+
+  protected createUnknown(wizard: WizardComponent, navigationModeInput: string): NavigationMode {

```

Comment:
Why did you override `createUnknown` instead of `createByName`?
I mean the navigation mode isn't unknown to the factory right?

Reply:
Right. Fixed in fb0b142.